scripts/local_es_setup.sh

```diff
@@ -14,7 +14,7 @@ curl -XPUT ${1}:9200/_template/template_foxtrot_mappings -d '
     },
     "mappings" : {
         "document" : {
-            "_source" : { "enabled" : false },
+            "_source" : { "enabled" : true },
```

Comment:
No.. We do not store source in ES. That's why we have hbase. Also, this makes the ES magnitudes heavier.